draft-gpew-priv-ppm.md

```diff
@@ -547,55 +564,45 @@ This message is called the client's *report*. It contains the following fields:
   the order of the task's `aggregator_endpoints` (i.e., the first share should
   be the leader's, the second share should be for the first helper, and so on).
 
-Encrypted input shares are structured as follows:
-
-~~~
-struct {
-  HpkeConfigId aggregator_config_id;
-  opaque enc<1..2^16-1>;
-  opaque payload<1..2^16-1>;
-} EncryptedInputShare;
-~~~
-
-* `aggregator_config_id` is equal to `HpkeConfig.id`, where `HpkeConfig` is the
-  key config of the aggregator receiving the input share.
-* `enc` is the HPKE encapsulated key, used by the aggregator to decrypt its
-  input share.
-* `payload` is the encrypted input share.
-
 To generate the report, the client begins by sharding its measurement into a
 sequence of input shares as specified by the VDAF in use. To encrypt an input
 share, the client first generates an HPKE {{!I-D.irtf-cfrg-hpke}} context for
 the aggregator by running
 
 ~~~
-enc, context = SetupBaseS(pk,
-                          "ppm input share" || task_id || server_role)
+enc, context = SetupBaseS(pk, Report.task_id || "ppm-01 input share" ||
```

Comment:
Assuming this gets adopted by the WG, the first draft will be "draft-ietf-ppm-00" (or something like that). If you take this suggestion, don't forget to apply it below.

```suggestion
enc, context = SetupBaseS(pk, Report.task_id || "ppm-00 input share" ||
```